frb_codegen/src/generator/dart/mod.rs

```diff
@@ -104,12 +108,16 @@ fn get_dart_api_spec_from_ir_file(ir_file: &IrFile, block_index: BlockIndex) ->
     let dart_funcs = ir_file
         .funcs
         .iter()
-        .map(generate_api_func)
+        .map(|f| generate_api_func(f, ir_file))
         .collect::<Vec<_>>();
     let dart_structs = distinct_types
         .iter()
-        .map(|ty| TypeDartGenerator::new(ty.clone(), ir_file).structs())
+        .map(|ty| {
+            { TypeDartGenerator::new(ty.clone(), ir_file, Some(dart_api_class_name.to_string())) }
```

Comment:
nit: why `{}`

Reply:
I don't know, the formatter adds them automatically